src/php/php-agent-download.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+libc=$1
+version=$2
+
+if [ -z "$libc" ]; then
+  echo "'libc' argument not supplied. Defaulting to 'gnu'"
+  libc="gnu"
+elif [[ ! $libc =~ ^(musl|gnu)$ ]]; then
+  echo "error: invalid 'libc' argument provided. Exiting."
```

Comment:
Reporting the provided libc arg would aid troubleshooting.

Reply:
This script does not get executed by the end user. This is only executed as part of the build+release process that occurs on our end to generate the initcontainer that is uploaded to docker and can be pulled down by the agent operator.

Reply:
It would be useful to understand why the script failed in any use case.

Reply:
added in a04f93cd356b604de0f5b1327fbe8c8b7ef69562